src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -478,7 +535,7 @@ def run_check(self, ctx: AnalyzeContext) -> CheckResultData:
 
             justification.append("Found provenance in release assets:")
             justification.extend(
-                [asset.name for asset in provenance_assets],
+                [asset.url for asset in provenance_assets],
```

Comment:
I wonder  is the reason the asset's URL is displayed here instead of its artifact id, Behnaz?

Reply:
I assume with NPM attestation support added, displaying the URL to the attestation instead of the attestation name makes sense because NPM attestations are not files (like what we were dealing with previously) but HTTP response payload. `asset.name` here was always the name of the attestation file being downloaded but it is not applicable for the NPM attestation case.

I think URLs may even be the overall better output type here because they point directly to the place where the attestation can be found. Just the file name may not provide enough information, especially now that we have multiple places where an attestation can be discovered.

@behnazh-w Let us know if you are thinking the same, or if there is anything you want to add.